packages/plugin-board/src/index.js

```diff
@@ -48,6 +47,21 @@ registerPlugin(`board`, Board, {
         extract(options) {
           return Promise.resolve(options.body);
         }
+      },
+      {
+        name: `encryptChannel`,
+        direction: `outbound`,
+
+        test(ctx, options) {
+          if (get(options, `service`) === `board` && has(options, `body.aclUrlLink`)) {
```

Comment:
I try to avoid heuristics that are based on the service you're talking to, but if that's the only option, you probably want to use `ctx.spark.device.isSpecificServiceUrl()`. That'll let you deal with cases where you passed a full board url to `spark.request()` rather than a service/resource pair